src/utils/types.ts

```diff
@@ -0,0 +1,35 @@
+export interface FirebasePlayerInfo {
```

Comment:
```suggestion
export interface FirebasePlayerInfoHash {
```

Reply:
To keep it in line with the following names

Reply:
I think we should have it like this:

```ts
export interface FirebasePlayerInfo {
  nickname: string
  token: string
}

export interface FirebasePlayerInfoWithTurn {
  nickname: string
  token: string
  turn: number
}

export interface FirebasePlayerInfoWIthCards {
  token: string
  nickname: string
  card1: string
  card2: string
}
```

Reply:
Done